addons/website_slides/static/tests/tours/slides_tour_tools.js

```diff
@@ -322,12 +316,85 @@ var addNewCourseTag = function (courseTagName, backend) {
 }];
 };
 
+/**
+ * Adds a new quiz to the given section name.
+ *
+ * @param {string} sectionName
+ * @param {{name: string, questions: {title: string, options: {title: string, isCorrect: boolean}[]}[]}} quizConfig
+ * @param {boolean} backend
+ */
+const addQuizToSection = (sectionName, quizConfig, backend) => {
+    const prefix = backend ? ':iframe ' : '';
+    let quizSteps = [
+    ...addSectionContent(prefix, sectionName),
+{
+    content: 'eLearning: click on quiz',
+    trigger: prefix + 'a[data-slide-category=quiz]',
+    run: "click",
+}, {
+    content: 'eLearning: fill quiz title',
+    trigger: prefix + 'input[name=name]',
+    run: `edit ${quizConfig.name}`,
+}, {
+    content: 'eLearning: fill quiz completion time',
+    trigger: prefix + 'input[name=duration]',
+    run: "edit 10",
+}, {
+    content: 'eLearning: create and publish slide',
+    trigger: prefix + 'footer.modal-footer button:contains("Publish")',
+    run: "click",
+}];
+const addQuestionSteps = quizConfig.questions.flatMap(({ title, options }, questionIdx, questions) => {
```

Comment:
Could you please extract a function `addQuizQuestions(questions)` that returns this array of steps, so that we can use the similar structure as in other `addXToSection` functions (and only have the weird indentation here)?
```js
    const prefix = backend ? ':iframe ' : '';
    return [
...addSectionContent(prefix, sectionName),
... // other steps to create the quiz (currently in quizSteps)
...addQuizQuestions(quizConfig.questions, prefix),
];
```